src/main/java/org/folio/des/scheduling/quartz/ScheduledJobsRemover.java

```diff
@@ -0,0 +1,49 @@
+package org.folio.des.scheduling.quartz;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.des.exceptions.SchedulingException;
+import org.quartz.JobKey;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.quartz.impl.matchers.GroupMatcher;
+
+@Log4j2
+@RequiredArgsConstructor
+public class ScheduledJobsRemover {
+  private final Scheduler scheduler;
+  private List<String> exportTypes;
+
+  public ScheduledJobsRemover(List<String> exportTypes, Scheduler scheduler) {
```

Comment:
do you need this constructor if you have RequiredArgsConstructor annotation?

Reply:
I removed constructer and make final List<> exportTypes